.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Should this be checked in, or is it user specific?

Reply:
I think we can check it in for other VSCode users to run the Debugger. Should be the same for everyone with the VSCode NodeJS debugger installed.